client/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:xenial
+
+RUN apt-get update && apt-get install -y curl
+
+COPY torture.sh /torture.sh
```

Comment:
I would also avoid a custom image here and simply mount a configmap and shell it. Avoid the bareer of having a registry

Reply:
good idea, on it!

Reply:
done as well, please check it out